pkg/cmd/prune-backups/prune.go

```diff
@@ -33,10 +33,14 @@ type PruneOpts struct {
 	RetentionType      string
 	MaxNumberOfBackups int
 	MaxSizeOfBackupsGb int
+	BackupPath         string
 }
 
 func NewPruneCommand() *cobra.Command {
-	opts := PruneOpts{RetentionType: "None"}
+	opts := PruneOpts{
+		RetentionType: "None",
+		BackupPath:    BasePath,
```

Comment:
let's make this a cmdline arg of the prune opts, you can still default to BasePath

Reply:
and make `BasePath` a constant then, I think we can get away without the unit tests changing the path as a variable

Reply:
done 👍🏽